lib/src/default_emoji_picker_view.dart

```diff
@@ -21,7 +19,7 @@ class _DefaultEmojiPickerViewState extends State<DefaultEmojiPickerView>
   TabController? _tabController;
   OverlayEntry? _overlay;
   late final _scrollController = ScrollController();
-  late final _utils = EmojiPickerInternalUtils();
+  late final _utils = EmojiPickerUtils();
```

Comment:
Nice 👍  will make it bit easier for people who copy & paste the code for custom View